test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/JsonLight/AsyncBatchRoundtripJsonLightTests.cs

```diff
@@ -242,5 +1216,17 @@ private void ClientReadAsyncBatchResponse(byte[] responsePayload, string batchCo
                 }
             }
         }
+
+        private byte[] ConvertStringToByteArray(string content)
+        {
+#if NETCOREAPP1_0
+            var stream = new MemoryStream(System.Text.Encoding.GetEncoding(0).GetBytes(content));
+#else
+            var stream = new MemoryStream(System.Text.Encoding.Default.GetBytes(content));

```

Comment:
Did explicit encoding such as ASCII or UTF8 work, or your environment didn't exercise this alternative?

Reply:
Thanks for pointing this out, I forgot to change it. I changed it to use System.Text.Encoding.ASCII.GetBytes and it also works with .NET Core, so I am going to remove the #if. The other comment above will not longer apply.